packages/realm/src/Collection.ts

```diff
@@ -128,9 +128,9 @@ export abstract class Collection<
    * @note Adding the listener is an asynchronous operation, so the callback is invoked the first time to notify the caller when the listener has been added.
    * Thus, when the callback is invoked the first time it will contain empty arrays for each property in the `changes` object.
```

Comment:
We should add a `@param` to the jsdoc for `keyPaths` on all listeners that have implemented it.

Reply:
Yep - I must have forgotten that 😊